packages/flutter_tools/lib/src/commands/drive.dart

```diff
@@ -69,6 +69,27 @@ class DriveCommand extends RunCommandBase {
       ..addFlag('build',
         defaultsTo: true,
         help: 'Build the app before running.',
+      )
```

Comment:
These tool changes will need tests. There should be some existing examples in flutter_tools/tests/../commands.shard

Reply:
Test updated. I have put them within an existing test.